annotation/src/main/kotlin/co/selim/goldfinch/annotation/EventBusService.kt

```diff
@@ -2,4 +2,10 @@ package co.selim.goldfinch.annotation
 
 @Retention(AnnotationRetention.SOURCE)
 @Target(AnnotationTarget.CLASS)
-annotation class GenerateProperties
+annotation class GenerateProperties(
+  val visibility: Visibility = Visibility.PUBLIC
+)
+
+enum class Visibility {
+  PUBLIC, INTERNAL
```

Comment:
Just a thought, but why not add a `MATCH_CLASS` as a default instead? :P

Reply:
I added an `INHERIT` option. Will be included in next release 🙂